Sources/protoc-gen-swiftgrpc/Generator.swift

```diff
@@ -118,6 +131,9 @@ class Generator {
       if options.generateTestStubs && options.generateNIOImplementation {
         fatalError("Generating test stubs is not yet supported for SwiftGRPC-NIO.")
       }
+      if options.generateTestStubs && !options.generateImplementations {
+        fatalError("Generating server test stubs without an implementation is not yet supported.")
```

Comment:
Do we plan to support this? If not, maybe remove "yet"

Reply:
Removed for now.